src/app/compiler-worker.js

```diff
@@ -12,25 +13,22 @@ module.exports = function (self) {
 				compileJSON = null;
 
 				importScripts(data.data);
-				version = Module.cwrap("version", "string", []);
-				if ('_compileJSONCallback' in Module)
-				{
-					compileJSONInternal = Module.cwrap("compileJSONCallback", "string", ["string", "number", "number"]);
-					var missingInputCallback = Module.Runtime.addFunction(function(path) {
-						missingInputs.push(Module.Pointer_stringify(path));
-					});
-					compileJSON = function(input, optimize) {
-						return compileJSONInternal(input, optimize, missingInputCallback);
-					};
+
+				var compiler = solc(Module);
```

Comment:
Loading an emscripten compiled binary in the global context will be assigned to the `Module` variable.

`importScripts(data.data);` above will load the url and put it into the global context